apps/sumologic/src/connectors/users.ts

```diff
@@ -0,0 +1,90 @@
+import { z } from 'zod';
+import { SumologicError } from './commons/error';
+
+const sumologicUserSchema = z.object({
+  id: z.string(),
+  firstName: z.string(),
+  lastName: z.string(),
+  email: z.string(),
+  isActive: z.boolean(),
+  isMfaEnabled: z.boolean(),
+});
+
+export type SumologicUser = z.infer<typeof sumologicUserSchema>;
+
+const sumologicResponseSchema = z.object({
+  data: z.array(z.unknown()),
+  next: z.string().nullable(),
+});
+
+export type GetUsersParams = {
+  accessId: string;
+  accessKey: string;
+  sourceRegion: string;
+  afterToken?: string | null;
+};
+
+export type DeleteUsersParams = {
+  userId: string;
+  accessId: string;
+  accessKey: string;
+  sourceRegion: string;
+};
+
+export const getUsers = async ({ accessId, accessKey, afterToken, sourceRegion }: GetUsersParams) => {
+  const encodedKey = Buffer.from(`${accessId}:${accessKey}`).toString('base64');
+
+  const url = new URL(`https://api.${sourceRegion.toLowerCase()}.sumologic.com/api/v1/users`);
```

Comment:
```suggestion
  const url = new URL(`https://api.${sourceRegion}.sumologic.com/api/v1/users`);
```
We don't need to lower the case if we make sure sourceRegion is always in lowercase.